src/apps/testapps/testBBox.c

```diff
@@ -246,6 +246,28 @@ SUITE(BBox) {
                  "bboxHexEstimate of invalid resolution fails");
     }
 
+    TEST(bboxHexEstimate_ratio) {
+        BBox bbox1 = {0.82294, 0.82273, 0.131671, 0.131668};
+        BBox bbox2 = {0.131671, 0.131668, 0.82294, 0.82273};
+        int64_t numHexagons1;
+        int64_t numHexagons2;
+
+        t_assert(bboxHexEstimate(&bbox1, 15, &numHexagons1) == 0,
+                 "should not fail");
+        t_assert(bboxHexEstimate(&bbox2, 15, &numHexagons2) == 0,
+                 "should not fail");
+
+        double diffPercentage = fabs(1.0 - numHexagons1 / (double)numHexagons2);
+
+        // numHexagons1 and numHexagons2 cannot be exactly equals because the
+        // diameter of the two bboxes is not exaxtly the same. (It's calculated
+        // using greatCircleDistanceKm)
```

Comment:
```suggestion
        // numHexagons1 and numHexagons2 cannot be exactly equal because the
        // diameter of the two bboxes is not exactly the same (it's calculated
        // using greatCircleDistanceKm)
```